lib/vary.js

```diff
@@ -11,9 +11,8 @@ function vary (field) {
     : String(value)
 
   // set new header
-  if ((value = append(header, field))) {
-    this.header('Vary', value)
-  }
```

Comment:
are you sure this if is not needed?

Reply:
we can get `false` in this condition only if the original function returns an empty string. Currently, `append`  from `jshttp/vary` handles all empty inputs and either throw a [new error](https://github.com/jshttp/vary/blob/master/index.js#L43) that handles in `fastify` or returns the correct string

Reply:
awesome, thanks!

Reply:
Thank you, for the great project and its ecosystem